### ISSUE TITLE
[Feature request] PodCoverageTests should accept file finder

### ISSUE THREAD

**Issue by van-de-bugger** [rjbs/Dist-Zilla#505]
`PodCoverageTests` plugin should accept file finder to let user specify files to tests.

Rationale: I have some modules which generated by 3-rd party plugins. These modules are out of my control so there is no meaning in checking them: anyway I cannot add POD to generated modules.

**Comment by karenetheridge** [comment]
I am currently revising the way filefinders are used in the [Test::EOL] plugin. Test generators in core can cargo-cult the same code once I'm done.

**Comment by rjbs** [comment]
Did this happen?

**Comment by karenetheridge** [comment]
nope, still on tuit list.  feel free to tag this appropriately :)

**Comment by dolmen** [comment]
Test::Pod::Coverage relies on Pod::Coverage which takes a package name (not a file path) and resolve it with `Pod::Coverage::_get_pods` which uses `Pod::Find::pod_where`. But it seems that `_get_pods` will also use a `pod_from` which is a path to a file and that can be given with `pod_coverage_ok($package, { pod_from => $file })`

**Comment by dolmen** [comment]
@van-de-bugger Have you tried [`[Test::Pod::Coverage::Configurable]`](https://metacpan.org/pod/Dist::Zilla::Plugin::Test::Pod::Coverage::Configurable)? It seems to have the features you want...

**Comment by van-de-bugger** [comment]
@dolmen: 

Not yet, I was not aware about `Test::Pod::Coverage::Configurable`.

Anyway, I would like to use something with file finder, because I already have a file finder which returns all *my* modules. In case of `Test::Pod::Coverage::Configurable` I will have to specify 3rd-party modules manually. It is not a good approach for a bundle.
